extension/persistence/eclipselink/src/main/java/io/polaris/extension/persistence/impl/eclipselink/PolarisEclipseLinkMetaStoreSessionImpl.java

```diff
@@ -124,7 +124,7 @@ private Map<String, String> loadProperties(String confFile, String persistenceUn
     }
 
     try {
-      InputStream input = this.getClass().getClassLoader().getResourceAsStream(confFile);
+      FileInputStream input = new FileInputStream(confFile);
```

Comment:
This is fine but Persistence.createEntityManagerFactory(persistenceUnitName, properties) puts the limitation which can only loads the configuration from a jar. I'm making a change to workaround that. https://github.com/polaris-catalog/polaris/pull/88

Reply:
I was able to fix this in the last commit.

Reply:
I replied in another thread.

@guitcastro Actually it's not because of the properties. Of course we need the properties because we want to replace realm in the database name - each realm will have its own database.

Persistence.createEntityManagerFactory() will still load configuration file from resource even without properties.

Reply:
@aihuaxu only if you specify the `ECLIPSELINK_PERSISTENCE_XML` property. I've tested and it worked. 

From their [docs](https://github.com/eclipse-ee4j/eclipselink/blob/43356e84b79e71022b1656a5462b0a72d70787a4/foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/config/PersistenceUnitProperties.java#L2575):

>  This property is only used by EclipseLink when it is locating the configuration file. 
> When used within an EJB/Spring container in container managed mode the locating and reading of this file is done by the container and will not use this configuration.

Reply:
I've added unit test to ensure config loads from outside the classpath is working.

Reply:
@guitcastro  Can you list the steps how you get it to work? It's possible the unit test found the persistence.xml from the embedded jar. That's why it doesn't fail for you.

Reply:
@aihuaxu Sorry, I have forgot to push it. Can you please take a look now? I also added a test to ensure that the config is not being loaded from the classpath.